ext/curl/interface.c

```diff
@@ -285,28 +285,14 @@ PHP_MINFO_FUNCTION(curl)
 			{"SSL",  CURL_VERSION_SSL},
 			{"SSPI",  CURL_VERSION_SSPI},
 			{"TLS-SRP", CURL_VERSION_TLSAUTH_SRP},
-#if LIBCURL_VERSION_NUM >= 0x072100 /* 7.33.0 */
 			{"HTTP2", CURL_VERSION_HTTP2},
-#endif
-#if LIBCURL_VERSION_NUM >= 0x072600 /* 7.38.0 */
 			{"GSSAPI", CURL_VERSION_GSSAPI},
-#endif
-#if LIBCURL_VERSION_NUM >= 0x072800 /* 7.40.0 */
 			{"KERBEROS5", CURL_VERSION_KERBEROS5},
 			{"UNIX_SOCKETS", CURL_VERSION_UNIX_SOCKETS},
-#endif
-#if LIBCURL_VERSION_NUM >= 0x072f00 /* 7.47.0 */
 			{"PSL", CURL_VERSION_PSL},
-#endif
-#if LIBCURL_VERSION_NUM >= 0x073400 /* 7.52.0 */
 			{"HTTPS_PROXY", CURL_VERSION_HTTPS_PROXY},
-#endif
-#if LIBCURL_VERSION_NUM >= 0x073800 /* 7.56.0 */
 			{"MULTI_SSL", CURL_VERSION_MULTI_SSL},
-#endif
-#if LIBCURL_VERSION_NUM >= 0x073900 /* 7.57.0 */
 			{"BROTLI", CURL_VERSION_BROTLI},
-#endif
 #if LIBCURL_VERSION_NUM >= 0x074001 /* 7.64.1 */
```

Comment:
https://github.com/curl/curl/blob/curl-7_43_0/include/curl/curlver.h#L74 version comparison macro can be used since curl 7.43

Reply:
It certainly seems to make it intuitive to use the human readable numbers compared the the hex comparison. 

Not all comparisons we have are `>=`; there are few `#if LIBCURL_VERSION_NUM <` comparisons, so even if we were to use `CURL_AT_LEAST_VERSION`, we will have a few `LIBCURL_VERSION_NUM` sprinkled everywhere. Perhaps it's good to use `LIBCURL_VERSION_NUM` comparisons for consistency?

Reply:
IMHO, we shouldn't increase your external dependencies too much. Aren't comments enough?

Reply:
There is no extra external dependency needed when version >= 7.61.0 will be required globally.

Reply:
Of course, that's true, but I'm also concerned that if this macro were to be deprecated or changed in the future, it would have a huge impact.

Reply:
`LIBCURL_VERSION_NUM` define is already used and is declared in the same curlver.h external header - the same can be said about it as well. IMO, `CURL_VERSION_BITS` should be fine to be used here.

Reply:
I don't think changing existing if checks is a good idea at all.

It will make merging up more difficult for no reason.